test/unit/util/AcceptParser.test.ts

```diff
@@ -0,0 +1,70 @@
+import { parseAccept, parseAcceptCharset, parseAcceptLanguage } from '../../../src/util/AcceptParser';
+
+describe('AcceptParser', (): void => {
```

Comment:
Because currently it is written as only a parser, not a validator. So invalid headers would cause unexpected output, which could either be an error or a accept value which doesn't make sense. I could look into extending it as a validator. Initially I thought it might make it too complex, but it might actually still be ok since everything gets split down to smaller parts.

Reply:
Yeah, should be doable I think. It's important though, since we don't want the server to crash when someone sense an invalid accept header.

Reply:
Well the server wouldn't crash. The user would simply get an error response back from the server, or the resource store would ignore their preferencs since it wouldn't match anything they know.

Reply:
Ah ok, didn't know there was a try-catch for this. In that case, the error handling is probably not that urgent. I would make an issue for it an defer.

Reply:
It's one of the parts of the BodyParser handler, which is part of the AuthenticatedLdphandler, which will catch the errors. So it's a bit further up the stack it gets caught.